docker-compose.yaml

```
@@ -5,4 +5,33 @@ services:
+      - prometheus
+
+  # Your Scala application
+  scala-app:
+    build: .
+    ports:
+      - "8080:8080"  # Adjust if your app uses a different port
+    environment:
+      - OTEL_EXPORTER_OTLP_ENDPOINT=http://jaeger:14250
+    depends_on:
+      - prometheus
+      - jaeger
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)